addon/components/o-s-s/star-rating.hbs

```diff
@@ -1,8 +1,8 @@
 <div class="star-rating fx-row">
-  {{#each this.activeStars}}
-    <OSS::Icon @style="solid" @icon="fa-star" class={{this.activeColorClass}} />
-  {{/each}}
-  {{#each this.passiveStars}}
-    <OSS::Icon @style="solid" @icon="fa-star" class={{this.passiveColorClass}} />
+  {{#each this.stars as |star index|}}
+    <OSS::Icon class="{{unless this.disablePointerCursor 'no-pointer-cursor'}} {{this.activeColorClass}}"
```

Comment:
given there is an `activeColorClass` getter, why not not move the `disablePointerCursor`  handling there?
alternatively, you can leave the `cursor` change to a `role="button"` attribute on the element, meaning you can have something like (and no need for custom CSS either for cursor: auto or pointer anymore) :
```
<OSS::Icon ... role={{unless this.disablePointerCursor "button"}} />
```

Reply:
the usage of role sounds good, the first part does not seem compatible though (I think). In any case we're getting active and passive stars, it's just that when `disablePointerCursor` is enabled we don't want the stars to be clickable :)

Reply:
@phndiaye that was what I did in the first place but we wanted to manage it from the parent & not make it dependent on the onChange argument to set it in the getter